src/irc/core/irc-session.c

```diff
@@ -90,6 +96,10 @@ static void sig_session_restore_server(IRC_SERVER_REC *server,
 	server->emode_known = config_node_get_bool(node, "emode_known", FALSE);
 	server->isupport_sent = config_node_get_bool(node, "isupport_sent", FALSE);
 
+	server->connrec->sasl_mechanism = config_node_get_int(node, "sasl_mechanism", SASL_MECHANISM_NONE);
+	server->connrec->sasl_username = g_strdup(config_node_get_str(node, "sasl_username", NULL));
+	server->connrec->sasl_password = g_strdup(config_node_get_str(node, "sasl_password", NULL));
```

Comment:
Strdup with no corresponding free? Maybe overwriting an existing value in connrec? (haven't checked)